Kang ji hyeock map 현재위치 정보 버튼 활성화

KangJiHyeock opened the pull request:
현재 위치에 마커가 생기고 카메라 이동

![image](https://github.com/user-attachments/assets/a12e50fa-aa06-471d-af20-2c2381a3140f)

이 버튼을 누르면

![image](https://github.com/user-attachments/assets/184c9027-28d9-4e80-bf41-27d01ff833ec)

이렇게 현재 위치로 카메라가 이동하고 마커가 생기게 만들었습니다

+ env파일도 추가하였습니다

KangJH02 committed:
Update map.dart 현재 위치정보

현재 위치정보를 geolocator을 이용하여 가져온 이후 location버튼을 누르면 현재 위치에 마커가 생기며, 카메라가 이동한다

KangJH02 committed:
Update AndroidManifest.xml geolocator을 사용하기 위한 정보입력

geolocator을 사용하기 위해 
<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
추가로 입력

KangJH02 committed:
Update pubspec.yaml geolocator추가

geolocator: ^13.0.2 추가

KangJH02 committed:
[Update] pubspec.yaml geolocator 추가

geolocator 추가

KangJH02 committed:
Create .env 

env파일 추가

seongjxn commented:
<img width="1228" alt="image" src="https://github.com/user-attachments/assets/83b18b38-c44e-4381-b87e-0bbb99fae7bd">

iOS 에서 GPS 권한 받아오는 부분이 수정이 안된 것 같습니다!

seongjxn commented:
아 그리고 현재 위치 버튼 아이콘은 지금 마커 아이콘보다는

<img width="74" alt="image" src="https://github.com/user-attachments/assets/0e7ba548-b5b3-440e-a1d9-5230db8ec865">

Icons.my_location

으로 변경하는 것이 더 좋아 보입니다!

KangJH02 committed:
[Update] ios geolocator 사용하기 위한 설정

KangJH02 committed:
[Update] 현재 위치 아이콘 수정, 현재 위치 수정

현재 위치버튼을 누르면 현재 위치로 계속 이동(5m간격으로)하도록 수정,
현재 위치가 이동하면 마커도 따라가도록 생성
현재 위치 아이콘 수정 및 on off 버튼 형식으로 제작

KangJiHyeock commented:
현재 위치 아이콘을 업데이트하였고, 버튼을 누르면 마커가 현재 위치를 따라가도록 만들었습니다. 또한 마커를 누르면 해당 마커에 대한 정보가 표시되도록 구현했습니다. 하지만 마커를 제거해도 정보 창이 계속 보이는 문제가 있어 현재 수정중입니다!

seongjxn commented:
방금 확인해봤는데, info.plist 수정하시면서 앱 이름이랑 전부 바뀐거같습니다!

확인해보시고 development에 최신 커밋에 있는 info.plist 기반으로 다시 수정 가능하실까요?

KangJH02 committed:
[Update] Info.plist 이름 수정

KangJH02 committed:
[Update] Info.plist 찐막

KangJH02 committed:
[Update] map.dart 지도 부분을 눌렀을 때 화면에 표시된 마커 사라짐 문제 해결

KangJiHyeock commented:
늦어서 죄송합니다.
ios 앱 이름 변경 문제 해결하였습니다
+ 4번 지도 아무부분을 누르면 화면에 표시된 마커가 사라지는 문제도 해결하였습니다.
나머지 문제들도 빠른 시일내에 해결하겠습니다

seongjxn commented:
헉 방금 확인했는데, 
.env 파일은 Git에 올리면 안됩니다!

seongjxn committed:
[REMOVE] .env 파일 제거

seongjxn commented:
.env 파일 git에서 제거했습니다.
.gitignore 파일에 *.env 등록되어 있는데 아마 수동으로 올리신 것 같습니다!

다음부터는 주의해주세요

KangJiHyeock commented:
헉 죄송합니다..

seongjxn reviewed (approved):
문제 없을 것 같습니당
